public/css/main.css

```diff
@@ -1,13 +1,19 @@
+html {
+	height: 100%;
+}
+
 body {
-	/* margin: 0 auto; */
-	/* width: 600px; */
+	display: flex;
+		align-items: center;
+		justify-content: center;
```

Comment:
Well… I was kind of trying out something new - not sure how much I like it though. Basically the idea was to have properties that apply to another property (like `display: flex` in this case) to be "under" those properties.

On a more general note, our CSS needs some revamping too to make it more modular so it can be maintained more easily as it grows.